.github/workflows/ci.yml

```
@@ -17,6 +17,11 @@ jobs:
       - name: Install Rust toolchain
         uses: actions-rust-lang/setup-rust-toolchain@v1
         with:
+          toolchain: nightly
+        with:
```

actionlint (1.7.7)

key "with" is duplicated in element of "steps" section. previously defined at line:19,col:9

(syntax-check)